src/com/haskforce/cabal/highlighting/CabalAnnotator.scala

```diff
@@ -20,8 +20,7 @@ class CabalAnnotator extends Annotator {
 
   private def setHighlighting(element: PsiElement, holder: AnnotationHolder, key: TextAttributesKey) {
     holder
-      .newSilentAnnotation(HighlightSeverity.INFORMATION)
-      .enforcedTextAttributes(EditorColorsManager.getInstance.getGlobalScheme.getAttributes(key))
-      .create()
+        .createWeakWarningAnnotation(element, "Message")
+      .setEnforcedTextAttributes(EditorColorsManager.getInstance.getGlobalScheme.getAttributes(key))
```

Comment:
Is this change intended to be here?

Reply:
yes as it  does not compile otherwise

Reply:
Strange, those changes fail to compile for me due to their deprecation -

```
:compileScala
.../intellij-haskforce/src/com/haskforce/cabal/highlighting/CabalAnnotator.scala:23: method createWeakWarningAnnotation in trait AnnotationHolder is deprecated: see corresponding Javadoc for more information.
        .createWeakWarningAnnotation(element, "Message")
         ^
one error found
```

Reply:
should be removed now